static/src/javascripts/projects/admin/bootstraps/commercial-browser-performance.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+/* global $, Epoch*/
+import config from 'lib/config';
+import fetchJson from 'lib/fetch-json';
+import template from 'lodash/utilities/template';
+import countBy from 'lodash/collections/countBy';
+import find from 'lodash/collections/find';
+
+// Globals that aren't imported
+declare var $: any;
+declare var Epoch: any;
```

Comment:
I can't track down where these come from so couldn't easily convert them to import statements.